.markdownlintignore

```diff
@@ -1,9 +1,9 @@
-# Ignore build directory
-/Build/
-
-# Ignore external dependencies
-*_extdep/
-
-# Ignore cloned dependencies
-/Common/MU_TIANO
+# Ignore build directory
+/Build/
+
+# Ignore external dependencies
+*_extdep/
+
+# Ignore cloned dependencies
+/Common/MU_TIANO
 /MU_BASECORE
```

Comment:
Since you're already making a line ending only change, can you please add an empty line to the end of any files in the PR missing one?

Reply:
Updated.